site/layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" data-bs-theme="auto">
```

Comment:
We might revert this since the default could be dark and we didn't change our dark mode !

Reply:
Yeah depending on how we handle the theme switcher in the future. For now it's mandatory until we drop the theme switcher.